src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  setCurrent: (newTodo: Todo) => {},
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  setCurrent,
+}) => {
+  return (
+    <tr data-cy="todo">
+      <td className="is-vcentered">{todo.id}</td>
+      <td className="is-vcentered">
+        {todo.completed
+          && (
+            <span className="icon" data-cy="iconCompleted">
+              <i className="fas fa-check" />
+            </span>
+          )}
+      </td>
+
+      <td className="is-vcentered is-expanded">
+        <p className={cn({
+          'has-text-danger': !todo.completed,
+          'has-text-success': todo.completed,
+        })}
+        >
+          {todo.title}
+        </p>
+      </td>
+
+      <td className="has-text-right is-vcentered">
+        <button
+          data-cy="selectButton"
+          className="button"
+          type="button"
+          onClick={() => setCurrent(todo)}
+        >
+          <span className="icon">
+            <i className="far fa-eye" />
+          </span>
+        </button>
+      </td>
```

Comment:
There are 2 types of fa-eye icons. `fa-eye` when there is no selected todo and `fa-eye-slash` when there is a selected todo